test/unit/EncryptionUtil.test.js

```diff
@@ -0,0 +1,171 @@
+import assert from 'assert'
+import crypto from 'crypto'
+
+import { ethers } from 'ethers'
+import { MessageLayer } from 'streamr-client-protocol'
+
+import EncryptionUtil from '../../src/EncryptionUtil'
+
+const { StreamMessage } = MessageLayer
+
+describe('EncryptionUtil', () => {
+    it('rsa decryption after encryption equals the initial plaintext', () => {
+        const encryptionUtil = new EncryptionUtil()
+        const plaintext = 'some random text'
+        const ciphertext = EncryptionUtil.encryptWithPublicKey(Buffer.from(plaintext, 'utf8'), encryptionUtil.getPublicKey())
+        assert.deepStrictEqual(encryptionUtil.decryptWithPrivateKey(ciphertext).toString('utf8'), plaintext)
```

Comment:
Should use `jest`'s `expect` rather than `assert`: https://jestjs.io/docs/en/expect